bin/node/cli/src/chain_spec.rs

```diff
@@ -355,6 +354,13 @@ pub fn testnet_genesis(
 		grandpa: GrandpaConfig { authorities: vec![] },
 		technical_membership: Default::default(),
 		treasury: Default::default(),
+		updater: UpdaterConfig {
+			phantom: Default::default(),
+			members: vec![AccountId::from_ss58check("5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY").unwrap() /// ALICE,
+						AccountId::from_ss58check("5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty").unwrap() /// BOB,
+						AccountId::from_ss58check("5FLSigC9HGRKVhB9FiEo4Y3koPsNmBmLJbpXg2mp1hXcS59Y").unwrap() /// CHARLIE,	
+						AccountId::from_ss58check("5DAAnrj7VHTznn2AWBemMuyBwZWs6FNFjdyVXUeYum3PTXFy").unwrap() /// DAVE,		],
+		},
```

Comment:
`///` is for documentation, `//` are regular comments.
No commas in the array, no closing of `[`.
This code breaks, a clear sign that a `cargo build` command has not been run.